tests/30rooms/60version_upgrade.pl

```diff
@@ -547,6 +462,96 @@ sub upgrade_room_synced {
       });
    };
 
+test "/upgrade preserves the power level of the upgrading user in old and new rooms",
```

Comment:
Why move this test?

Reply:
It looks better lower down the file :D 

(just an accident)